Sources/AblyChat/AblyCocoaExtensions/Ably+Dependencies.swift

```diff
@@ -9,10 +9,14 @@ import Ably
     extension ARTRealtimeChannels: RealtimeChannelsProtocol, @retroactive @unchecked Sendable {}
 
     extension ARTRealtimeChannel: RealtimeChannelProtocol, @retroactive @unchecked Sendable {}
+
+    extension ARTRealtimePresenceQuery: @retroactive @unchecked Sendable {}
```

Comment:
As mentioned in ably/ably-cocoa#1973, `ARTRealtimePresenceQuery` isn't thread-safe so we can't just mark it as `Sendable`. Please can you explain the problem you were trying to solve?

Reply:
Without it we have this (in `MockPresence`):
![Screenshot 2024-09-26 at 01 23 22](https://github.com/user-attachments/assets/c5675381-2536-4075-a495-3a9d37fbc9f8)

Reply:
See #65

Reply:
Didn't add it after https://github.com/ably-labs/ably-chat-swift/pull/34/commits/24f15e522716486a2ebf4e5bb70cb1d0ea25ae69